cmds/boot/localboot/grub.go

```diff
@@ -177,15 +177,12 @@ func isValidFsUUID(uuid string) bool {
 	return true
 }
 
-func unquote(ver grubVersion, text string) string {
-	if ver == grubV2 {
-		// if grub2, unquote the string, as directives could be quoted
-		// https://www.gnu.org/software/grub/manual/grub/grub.html#Quoting
-		// TODO unquote everything, not just \$
-		return strings.Replace(text, `\$`, "$", -1)
-	}
-	// otherwise return the unmodified string
-	return text
+func unquote(text string) string {
```

Comment:
Can we name it sth like unquoteCmdline, so that people don't use it to unquote other random stuff? I would appreciate some of the description as part of the commit msg too

Reply:
Good idea. I named it unquoteGrubString since it can be (and is) used for other GRUB-related strings. I elaborated in the commit message, too.